config/locales/mailers/applicants.en.yml

```diff
@@ -0,0 +1,7 @@
+en:
+  mailers:
+    applicant_mailer:
+        assignment:
+          subject: "%{application_name}: Assigned as applicant on %{title}"
+        unassignment:
+          subject: "%{application_name}: Unassigned as applicant on %{title}"
```

Comment:
If you change the name of the mailers, they need to be changed here.

I think "Added as an applicant to" and "Removed from" may be better subjects.

Reply:
I like the idea of keeping this uniform between reviewers and applicants and I think the language is clear enough here.